src/List/list.store.js

```diff
@@ -63,15 +68,22 @@ export default Store.create({
                 error(`${modelType} is not a valid schema name`);
             }
 
-            let modelDefinition = d2.models[modelType];
-
+            let modelDefinition;
             if (searchString) {
-                modelDefinition = d2.models[modelType].filter().on('displayName').ilike(searchString);
+                modelDefinition = d2.models[modelType]
+                    .filter().on('displayName').ilike(searchString)
+                    .filter().on('userCredentials.username').ilike(searchString);
+            } else {
+                modelDefinition = d2.models[modelType]
+                    .filter().on('name').notEqual('default');
             }
 
             const listSearchPromise = modelDefinition
-                .filter().on('name').notEqual('default')
-                .list({ fields: fieldFilteringForQuery });
+                .list({
```

Comment:
No, this refactor is required because now the search term looks in both displayName and userCredentials.username. It's a bit tricky: when the search was enabled, we were doing "search(displayName) AND name != "default". But now we need: "(search(displayName) OR search(username)) AND name != "default"). That's not supported in a single call, all ANDs or all ORs. So name != "default" is not done on a search. When no search term is given, it does filter out the user default. In any case, there is no user default in their database, I am really sure the origin of that filter.